kover-gradle-plugin/docs/index.md

```diff
@@ -36,11 +42,638 @@ Gradle plugin that measures the coverage for tests running on the JVM and genera
 * Using JaCoCo library as an alternative for coverage measuring and report generation.
 
 ## Quickstart
+To make the minimal configuration of the plugin, determine the type of your Gradle build and learn the instructions for configuring
+
+## Build types
+### Single module Kotlin JVM project
+Please refer to the [example project](https://github.com/Kotlin/kotlinx-kover/tree/main/kover-gradle-plugin/examples/jvm/single).
+
+#### Application of Kover plugin
+[Apply the plugin](#apply-kover-gradle-plugin-in-project) in a single project in the Gradle build.
+
+After applying Kover Gradle plugin, during the running tests, the classes are modified (instrumented) when loaded into the JVM which may lead to some performance degradation, or affect concurrent tests.
```

Comment:
> which may lead to some performance degradation, or affect concurrent tests.

This is rather scary description of what is happening — I doubt regular user will notice problems with instrumented tests. Such sentence may make some users afraid of using Kover. I would rather just cut this part at all and/or replace it with 'to learn more about instrumentation and how to solve typical issues you may have with it, refer to corresponding section'

Reply:
We can soften the wording a little, but it's not worth removing it.
Because some users unexpectedly experience a drop in performance, and if we delete this description or move it somewhere far away, they will not know in advance how instrumentation affects tests.

See: https://github.com/Kotlin/kotlinx-kover/issues/531

Reply:
WDYT of:

After applying Kover Gradle plugin, the classes are modified (instrumented) when loaded into the JVM during regular `test` runs, so number of line hits can be counted. This may impact your testing experience. To learn more about instrumentation, it's configuring, and solutions for typical problems, please refer to the [corresponded section](#instrumentation).